docs/src/snap/howto/security/disa-stig-assessment.md

```diff
@@ -0,0 +1,6187 @@
+# DISA STIG for {{product}}
+
+Security Technical Implementation Guides (STIGs) are developed by the Defense
+Information System Agency (DISA) for the U.S. Department of Defense (DoD).
+
+The Kubernetes STIG contains guidelines on how to check and remediate various
+potential security concerns for a Kubernetes deployment.
+
+{{product}} aligns with many DISA STIG compliance recommendations by default.
+However, additional hardening steps are required to fully meet the standard.
+
+## What you'll need
+
+This guide assumes the following:
+
+- You have a bootstrapped {{product}} cluster (see the [getting started] guide)
+- You have root or sudo access to the machine
+- You have reviewed the [post-deployment hardening] guide and have applied the
+  hardening steps that relevant to your use-case 
+
+
+## Results overview
+
+| Status | Findings |
+| ------ | ----- |
+| `Automated` (70) | V-242379, V-242380, V-242381, V-242382, V-242387, V-242388, V-242389, V-242391, V-242392, V-242397, V-242400, V-242405, V-242406, V-242407, V-242408, V-242409, V-242418, V-242419, V-242420, V-242421, V-242422, V-242423, V-242426, V-242427, V-242428, V-242429, V-242430, V-242431, V-242432, V-242433, V-242434, V-242436, V-242444, V-242445, V-242446, V-242447, V-242448, V-242449, V-242450, V-242451, V-242452, V-242453, V-242456, V-242457, V-242459, V-242460, V-242466, V-242467, V-245542, V-245543, V-245544, V-254801, V-242376, V-242377, V-242378, V-242384, V-242385, V-242390, V-242402, V-242403, V-242404, V-242424, V-242425, V-242438, V-242461, V-242462, V-242463, V-242464, V-242465, V-245541 |
+| `Not Applicable` (13) | V-242386, V-242393, V-242394, V-242395, V-242396, V-242398, V-242399, V-242413, V-242437, V-242442, V-242443, V-242454, V-242455 |
+| `Manual` (8) | V-242383, V-242410, V-242411, V-242412, V-242414, V-242415, V-242417, V-254800 |
+
+**Automated**: An automated process can be used to validate that the system is
+correctly configured.
+
+**Not Applicable**: These Findings are not applicable to {{product}}. Some
+reasons for this include: a check on a Kubernetes feature that was removed
+prior to {{product}}'s first release, a check for a component that {{product}}
+does not package, etc.
+
+**Manual**: These checks require manual intervention from a cluster
+administrator, so they cannot be automated.
+
+
+### [V-242381]: The Kubernetes Controller Manager must create unique service accounts for each work payload
+
```

Comment:
I don't think we'll have any good way around this lint check if we insist on having the Finding's original `title` field in the section title here.

I'll look if there's a way to use `<!-- ignore -->` comments or something and add them to the report generation if I do.